src/components/1.Home/FilterAndSort.jsx

```diff
@@ -17,22 +21,50 @@ export default function FilterAndSort() {
 
     if (isLoading) return <h3>Items loading ...</h3>;
 
```

Comment:
Currently there are multiple 'items loading' messages as you have one from each component - not a huge issue, but I'd consider having a piece of state that displays a single loading message until all items have loaded, or a specific message to say which component is loading eg 'articles loading'